content/en/dashboards/widgets/geomap.md

```diff
@@ -63,7 +63,7 @@ The geomap widget visualizes geographic data with shaded regions or points. It c
   {{% /tab %}}
   {{< /tabs >}}
 
-3. Optional: Configure your view box to specify where you want to focus the map initially.
+3. (Optional) Under **Visual Options**, set the widget default view box. You can choose a custom view, or search for a country, subdivision (province, state) or city to define as the default view. A **Custom** view box can be any region, not specific to a state.
```

Comment:
@estherk15 I think I misspoke in the Jira ticket - it should be country or subdivision only, we don't allow for Cities yet to be defined as the default view.